config/environments/appliance.rb

```diff
@@ -6,7 +6,7 @@
 
 Rails.application.configure do
   config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new('conjur-possum'))
-  config.log_level = :info
+  config.log_level = ENV['CONJUR_LOG_LEVEL'] || :info
```

Comment:
Did you test to see what happens if a undefined log level value is provided here?  Is a reasonable error returned?

Reply:
nice one. I added a new PR that fixes this